docs/get-started/getting-started-alluxio.md

```diff
@@ -419,4 +419,13 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio limitations
+### Alluxio does not sync metadata from UFS(e.g. S3) by default
+Alluxio does not sync metadata from S3 by default, so it won't pick up changed files.
```

Comment:
I'd also like to see a sentence at the very beginning like line 19 that says something like:  sets up Alluxio to handle reads and by default does not handle file updates in the remote blob store. 

We should also explicitly put somethign in here about writes don't go through alluxio.

Reply:
Done